### DISCUSSION TITLE
RESULTS: NumFOCUS Board of Directors & Technical Steering Committee Election October-November 2024

### DISCUSSION THREAD

**Discussion by pavithraes** [numfocus/elections#85] [Announcements]
## Results ✨

I’m happy to share the results for the NumFOCUS Board of Directors & Technical Steering Committee Elections conducted in October-November 2024.

### Technical Steering Committee:

- Eloisa Elias T.
- Jacob Schreiber
- Juan Nunez-Iglesias
- Sanket Verma
- Sophia Yang
- Tetsuo Koyama
- Wolf Vollprecht

### Administrative Board:

- Andy Terrel
- Julie Krugler Hollek
- Mridul Seth

Congratulations to the elected members! 🎉

Thank you to everyone who participated in this election. I also thank you for your patience, it’s been a long and difficult process.

### Voting members 

147 people voted in this election, out of the 200 people who registered to vote. [^1]
[^1]: This is a 73.5% turnout. Including extension, the voting phase was approximately 3 weeks.

Among the voters, we had the following groups represented:

- ~65% voters represented 46 unique Sponsored projects
- ~14 % voters represented 18 unique Affiliated projects
- ~17% voters were NumFOCUS Event chairs (like PyData events)
- ~4% voters were NumFOCUS Committee members

#### Technical Steering Committee

The community nominated 24 individuals, out of which 13 people confirmed their nomination. [^2][^3]

[^2]: Folks could nominate themselves and/or any community member. Hence, we asked everyone, but especially folks nominated someone else, to confirm their nomination and share a statement in their own words.

[^3]: There was an additional review of nominees by the Board, based on [specific criteria](https://github.com/numfocus/elections/blob/main/election-process.md#review-nominees). The Board accepted all nominations.

Votes received by nominees:

![Technical Steering Committee](https://github.com/user-attachments/assets/ce4b25cc-dfa3-4a98-bb5c-dec1f2c3d4e0)

#### Administrative Board

The community nominated 22 individuals, out of which 20 people confirmed their nomination. [^2][^3]

Votes received by nominees:

![Administrative Board](https://github.com/user-attachments/assets/ec213fb4-6414-4f98-8dde-bc3f68dd9886)

> [!NOTE]
> Thomas Wiecki decided to step down from board candidacy after the voting phase, so his votes have not been included.

_Edit: The plots have been updated. Thank you, @pradyunsg, for pointing out the discrepancy in the numbers. I reviewed and verified that the results are unchanged, even though the previous numbers were incorrect._

### Voting and weighting system

The project votes were weighted as 3 votes for each Sponsored project and 1 vote for each Affiliated project, independent of the number of people voting from a particular project. For example, if _n_ number of people voted from _my_awesome_sponsored_project_, each vote was weighted as _3/n_. Committee member and event chair votes were weighted as 1 vote per individual.

It’s worth noting that the results after calculating weighted votes were identical to the results generated from raw non-weighted votes.

We used [checkbox voting](https://opavote.com/methods/traditional-plurality-voting), where every voting individual could vote for up to 7 members for the Technical Steering Committee, and up to 3 members for the Administrative Board.

## What’s Next? 💬

If you have any questions about the results, please ask them in the [Q/A category](https://github.com/numfocus/elections/discussions/categories/q-a). We’re working on documenting all aspects of the election, including rationale for all the decisions made. We’ll also have a feedback form where you share your experience participating in the election and ideas to improve the process.

Once again, I thank you all for helping us run the first community-voted election for the NumFOCUS Board. 🌻

**Comment by logankilpatrick** [comment]
Incredible work by the election committee to make this happen in a transparent, fair, and community driven way. 

Big thanks to everyone who put their name or names or others they respect into the running for one of the boards.

So excited to see us take this important step and look forward to the next election cycle as well!

**Comment by aterrel** [comment]
Agreed! Pavithra, Tetsuo, and Kaylan THANK YOU for your efforts.

**Comment by InessaPawson** [comment]
A heartfelt thank you to the Election Committee for helping the NumFOCUS community to reach this important milestone. 

To the nominees who did not win this time, your willingness to serve is deeply appreciated. While elections may have winners, this community offers many opportunities for impactful leadership beyond the two newly formed governing bodies.

Congratulations to the elected members of the Administrative Board and Technical Committee. May your meetings be short, and your decisions be wise!
